### ISSUE TITLE
Undefined index: constraints into qgisVectorLayer

### ISSUE THREAD

**Issue by laurentj** [3liz/lizmap-web-client#1586]
### What is the bug?

In my logs, i got tons of notices like that:

```
Undefined index: constraints
	at: /opt/lizmap-web-client/lizmap/modules/lizmap/classes/qgisVectorLayer.class.php	74
	url: /index.php/lizmap/service/?repository=rep2&project=veloroutes_edit
	POST: array (   'repository' => 'rep2',   'project' => 'veloroutes_edit',   'module' => 'lizmap',   'action' => 'service:index',   'SERVICE' => 'WFS',   'VERSION' => '1.0.0',   'REQUEST' => 'GetFeature',   'TYPENAME' => 'segment_cyclable_edit',   'OUTPUTFORMAT' => 'GeoJSON',   'FEATUREID' => 'segment_cyclable_edit.8718',   'GEOMETRYNAME' => 'extent', )
```

i would like to know if the inexisting 'constraints' in the layer properties given to qgisVectorLayer, is something that could be occured, or if there is an issue elsewhere into lizmap?

### Environment

 - Lizmap version: 3.3.4
 - QGIS Server version: 3.4
 - OS:  linux
 - Lizmap admin information (users, logs…) stored into PostgreSQL

**Comment by rldhont** [comment]
it is something that could be occured

**Comment by laurentj** [comment]
okay, i will made a patch then

**Comment by laurentj** [comment]
Sorry, logs about this issue were generated until november 2019. In the same month, you did a fix about constraints :  cc27fff65ce75010bae83fb0ffa24290285203f3 . No more errors after updating the instance with Lizmap 3.3.2 in november. It seems it fixed the issue.
